packages/next/src/server/lib/incremental-cache/file-system-cache.ts

```diff
@@ -50,31 +50,33 @@ export default class FileSystemCache implements CacheHandler {
     this.experimental = ctx.experimental
     this.debug = !!process.env.NEXT_PRIVATE_DEBUG_CACHE
 
-    if (ctx.maxMemoryCacheSize && !memoryCache) {
-      if (this.debug) {
-        console.log('using memory store for fetch cache')
-      }
+    if (ctx.maxMemoryCacheSize) {
+      if (!memoryCache) {
```

Comment:
This change should be reviewed without whitespace. I updated this to be consistent with how it's structured in fetch-cache, to prevent debug mode from printing:

> not using memory store for fetch cache

Even though it was, because the memory cache had already been instantiated.